### PR TITLE
Add examples for list label callback

### PR DESCRIPTION
**Pull request by rabauss**
Finally, I want to add some examples for the list label callback
After handling with https://github.com/contao/contao/pull/2906 I realized that this is necessary

### CODE REVIEW

File: docs/dev/reference/dca/callbacks.md [hunk 1]
```diff
@@ -378,9 +378,75 @@ to add status icons.
 **return:** `array` Columns with labels
 {{% /expand %}}
 
+{{% expand "Example for tree view" %}}
+
```
Comment (fritzmg) [inline docs/dev/reference/dca/callbacks.md]:
```suggestion
```

File: docs/dev/reference/dca/callbacks.md [hunk 2]
```diff
@@ -378,9 +378,75 @@ to add status icons.
...
+        $icon = Image::getHtml('bundles/app/images/example.svg');
+
+        return $icon.sprintf(' %s <span class="tl_gray" style="margin-left:3px;">[%s]</span>', $row['title'], $row['name']);
+    }
+}
+```
+{{% /expand %}}
+
+{{% expand "Example for list view" %}}
+
```
Comment (fritzmg) [inline docs/dev/reference/dca/callbacks.md]:
```suggestion
```

File: docs/dev/reference/dca/callbacks.md [hunk 3]
```diff
@@ -378,9 +378,75 @@ to add status icons.
...
+        return $labels;
+    }
+}
+```
+{{% /expand %}}
+
 ***
 
 
+
```
Comment (fritzmg) [inline docs/dev/reference/dca/callbacks.md]:
```suggestion
```

**Review by fritzmg — CHANGES_REQUESTED** [review]
Looks good, thank you very much :). Just some minor nitpicky whitespace changes.

**Review by rabauss — COMMENTED** [review]
Thanks @fritzmg for CS!

**Review by fritzmg — APPROVED** [review]

**Comment by fritzmg** [comment]
Thank you @rabauss !

**Pull request merged**